simObj.py

```diff
@@ -1,52 +1,58 @@
 from action import Action
 from cellCodes import CellCodes
 
+
 class ISimObj:
+    # is keeping the last action necessary?
```

Comment:
we need to keep object state and last_action served that purpose. However its bugged for wait action.
object state represented by self.grab should be enough.